doc/source/whatsnew/v2.1.0.rst

```diff
@@ -130,8 +130,10 @@ Copy-on-Write improvements
 
 .. _whatsnew_210.enhancements.map_na_action:
 
-``map(func, na_action="ignore")`` now works for all array types
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+New :meth:`DataFrame.map` method and support for ExtensionArrays
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :meth:`DataFrame.map` been added and :meth:`DataFrame.applymap` has been deprecated. :meth:`DataFrame.map` has the same functionality as :meth:`DataFrame.applymap`, but the new name better communicates that this is the :class:`DataFrame` version of :meth:`Series.map` (:issue:`52353`).
```

Comment:
It seemed to me we were burying the lead here; wanted to make the additional method to DataFrame be at the top.